client/components/editor/index.vue

```diff
@@ -12,21 +12,33 @@
         @selected="selectElement"
         :repository="repository"
         :activity="activity"
+        :elements="isPublishedPreview ? changesSincePublish : elements"
```

Comment:
The way that the state is hydrated cannot work for custom containers, it needs to be significantly rewritten.